wpeverest/everest-forms
# Feature/confirm email

### All Submissions:






### Changes proposed in this Pull Request:


This PR is about the Confirm Email option for the Email field. Just like password field confirm password works it also works in the same way.

### How to test the changes in this Pull Request:

1. Edit or create a form. 
2. Drag the Email field and go the field options of Email field.
3. On Field options, you will see a checkbox named Email confirmation please enable it and verify it by frontend.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Confirm Email option on Email Field.
